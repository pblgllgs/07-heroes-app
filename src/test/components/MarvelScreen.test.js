const { shallow } = require("enzyme");
const { MarvelScreen } = require("../../components/marvel/MarvelScreen");

describe('pruebas en <MarvelScreen />', () => {
    test('comparar con snapshot', () => {
        const wrapper = shallow(<MarvelScreen />)
        expect(wrapper).toMatchSnapshot();
    })
})
