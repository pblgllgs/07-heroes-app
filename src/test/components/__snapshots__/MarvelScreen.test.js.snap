// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <MarvelScreen /> comparar con snapshot 1`] = `
<div>
  <h1>
    Marvel Heroes
  </h1>
  <hr />
  <HeroList
    publisher="Marvel Comics"
  />
</div>
`;
